src/components/Dropdown/props.ts

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
giusto una nota sul nome del file, in genere lo chiamiamo `<component_name>.props.ts`

Reply:
I've seen this done in all the components but I honestly find it weird since it is alreayd in the "component_name" directory